src/components/Page/Base.js

```diff
@@ -6,16 +7,25 @@ import Footer from "@/pesayetu/components/Footer";
 import Navigation from "@/pesayetu/components/Navigation";
 import { navigationArgs } from "@/pesayetu/config";
```

Comment:
```suggestion
```
We should no longer need this.

Reply:
@kilemensi We do need it because of the select component propeerties...maybe not for the logoProps....but definitely for the selectProps

Reply:
We shouldn't need it @BrendaBucha ... This PR is supposed to pull everything Navigation needs from the CMS.

Reply:
@kilemensi Okay...so maybe a question would be how are we planning on handling our search? Or search data on wordpress. I am not sure this data needs to come from the primary menu

Reply:
Aren't we searching for locations (county/sub-county) @BrendaBucha ? They wouldn't be coming from WordPress. They'll be coming with the map. How are we loading the map?

Reply:
@BrendaBucha looking at the selectProps on the config, we can have the title ("Search for location") pulled together with the navigation.

Data API has this link that suggests geography, once you type a query with more than 3 words. https://v2.hurumap.org/api/v1/geography/search/3/?q=nai We can use this to populate the search menuItems. 

while the remaining 👇🏾 , I think this shouldn't be editable.
```
    selectId: "select-grouped-id",
    inputBaseId: "inputbase-grouped-id",
    selectLabel: "select-grouped-label",
    inputBaseLabel: "inputbase-grouped-label",
    openIcon: SearchOpen,
    closeIcon: SearchClose,
    ```